Makefile

```diff
@@ -37,7 +40,9 @@ clean:
 	@rm -f ubuntu-cleaner*.tar.gz
 
 test: virtualenv
-	$(PYTHON) -m unittest discover tests
+	$(COVERAGE) run --source=ubuntucleaner -m unittest discover tests
+	$(COVERAGE) report
+	@if [ -n "$$CI" ]; then $(COVERALLS); fi \
```

Comment:
This is specifically for the report to the thirdparty service coveralls.io
Some tokens and other variables are only available from within TravisCI hence we check if it's set so we only publish to coveralls.io when within the `CI`